### PR TITLE
Adding development container defination for NodeJS + Postgres

### LINKED ISSUE #42: Node + Postgres
Could you make a container template with node + postgres instead of mongodb?

### PR DESCRIPTION
**Pull request by kmehant**
Fixes 
#42

### CODE REVIEW

**Comment by kmehant** [comment]
@Chuxel Please review 😃

**Comment by Chuxel** [comment]
Apologies for the delayed response - we had enough contributions we wanted to lay down some regression testing, but I'll take a look at this soon.

File: containers/javascript-node-lts-postgres/test-project/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "docker web app",
```
Comment (Chuxel) [inline containers/javascript-node-lts-postgres/test-project/package.json]: This is an invalid name which is causing `npm install` to fail when you F5 the project.

File: containers/javascript-node-lts-postgres/.devcontainer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+FROM node:lts
+
+# Configure apt
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update \
```
Comment (Chuxel) [inline containers/javascript-node-lts-postgres/.devcontainer/Dockerfile]: We did some RUN layer consolidation around the time you submitted this PR, so could you grab the latest Dockerfile from javascript-node-lts and add it here?

File: containers/javascript-node-lts-postgres/test-project/package.json (hunk 1 above)
Comment (kmehant) [reply containers/javascript-node-lts-postgres/test-project/package.json]: @Chuxel Sorry for that! I have updated it now.

File: containers/javascript-node-lts-postgres/.devcontainer/Dockerfile (hunk 2 above)
Comment (kmehant) [reply containers/javascript-node-lts-postgres/.devcontainer/Dockerfile]: Yeah, @Chuxel, I have seen it and yes I have made those change now!

**Comment by Chuxel** [comment]
LGTM, thanks again for the contribution!

**Pull request merged**